src/discord-rest_ratelimit.c

```diff
@@ -20,8 +20,11 @@
 #define RATELIMITER_TABLE_FREE_VALUE(_value) free(_value)
 #define RATELIMITER_TABLE_COMPARE(_cmp_a, _cmp_b)                             \
     chash_string_compare(_cmp_a, _cmp_b)
-#define RATELIMITER_TABLE_INIT(route, _key, _value)                           \
-    memcpy(route.key, _key, sizeof(route.key));                               \
+#define RATELIMITER_MIN(a, b) ((a) < (b) ? (a) : (b))
```

Comment:
I think this macro doesn’t add much here, you can just use the ternary straight away.

Reply:
Alright, I'll do that. There ought to be a MIN macro for use everywhere in the codebase.